DigitalLearningSolutions.Data/Helpers/CourseHelper.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace DigitalLearningSolutions.Data.Helpers

```

Comment:
I really like separating these out from the big query, but I wonder if they still belong in the CourseDataService so the devs don't need to click between various files to read one query 🤔 If you prefer keeping it out I'd make sure to have references to "SQL"/"query" etc in the helper class and constants.